pkg/config/config.go

```diff
@@ -368,8 +369,8 @@ func InitConfig(config Config) {
 	// secrets backend
 	config.BindEnvAndSetDefault("secret_backend_command", "")
 	config.BindEnvAndSetDefault("secret_backend_arguments", []string{})
-	config.BindEnvAndSetDefault("secret_backend_output_max_size", secrets.SecretBackendOutputMaxSizeDefault)
-	config.BindEnvAndSetDefault("secret_backend_timeout", secrets.SecretBackendTimeoutDefault)
+	config.BindEnvAndSetDefault("secret_backend_output_max_size", 0)
+	config.BindEnvAndSetDefault("secret_backend_timeout", 30)
```

Comment:
Could we set those to 0 and let the secrets comp fallback on its default when configured with `0` ? That way all the secrets logic is kept in the component.